resources/js/components/attribute/Attribute.vue

```diff
@@ -0,0 +1,428 @@
+    import SingleChoice from '@/components/attribute/SingleChoice.vue';
+    import MultiChoice from '@/components/attribute/MultiChoice.vue';
+    import UserList from '@/components/attribute/UserList.vue';
+    import Url from '@/components/attribute/Url.vue';
+    import SqlAttr from '@/components/attribute/Sql.vue';
+    import SystemSeparator from '@/components/attribute/SystemSeparator.vue';
+    import DefaultAttr from '@/components/attribute/Default.vue';
+
+    export default {
+        components: {
+            'string-attribute': StringAttr,
+            'stringfield-attribute': Stringfield,
+            'richtext-attribute': Richtext,
+            'integer-attribute': IntegerAttr,
+            'float-attribute': FloatAttr,
+            'bool-attribute': Bool,
+            'percentage-attribute': Percentage,
+            'serial-attribute': Serial,
+            'dimension-attribute': Dimension,
+            'epoch-attribute': Epoch,
+            'list-attribute': List,
+            'tabular-attribute': Tabular,
+            'iconclass-attribute': Iconclass,
+            'rism-attribute': RISM,
+            'geography-attribute': Geography,
+            'entity-attribute': Entity,
+            'date-attribute': DateAttr,
+            'daterange-attribute': DaterangeAttr,
+            'singlechoice-attribute': SingleChoice,
+            'multichoice-attribute': MultiChoice,
+            'userlist-attribute': UserList,
+            'url-attribute': Url,
+            'sql-attribute': SqlAttr,
+            'system-separator-attribute': SystemSeparator,
+            'default-attribute': DefaultAttr,
+        },
+        props: {
+            data: {
+                type: Object,
+                required: true,
+            },
+            valueWrapper: {
+                type: Object,
+                required: false,
+                default: _ => new Object(),
+            },
+            disabled: {
+                type: Boolean,
+                required: false,
+                default: false,
+            },
+            reactTo: {
+                type: Number,
+                required: false,
+                default: -1,
+            },
+            hideLinks: {
+                type: Boolean,
+                required: false,
+                default: false,
+            },
+            preview: {
+                type: Boolean,
+                required: false,
+                default: false,
+            },
+            previewData: {
+                type: Object,
+                required: false,
+                default: _ => new Object(),
+            },
+        },
+        emits: ['change', 'update-selection'],
+        setup(props, context) {
+            const {
+                data,
+                valueWrapper,
+                disabled,
+                reactTo,
+                hideLinks,
+                preview,
+                previewData,
```

Comment:
Those are not used.

Reply:
These are some remains from stuff I copy&wasted 😅 can be deleted 👍